### PR TITLE
added image aspect ration preserving macro

### PR DESCRIPTION
**Pull request by N0taN3rd**
This pr adds a macro that will preserve an images aspect ratio when changing its dimensions

### CODE REVIEW

File: custommacros.sty [hunk 1]
```diff
@@ -14,3 +15,10 @@
   \InputIfFileExists{#1/\jobname.tmp}{}
   \AtEndDocument{\app@exe{rm -f #1/\jobname.tmp}}}
 \makeatother
+
+%%% This macro is a drop in replacement for \includegraphics that ensures the images 
+%%% apect ration is maintained when changing its width, height or scale
```
Comment (ibnesayeed) [inline custommacros.sty]: `apect ration` typo?

File: chapters/01_introduction.tex [hunk 2]
```diff
@@ -6,7 +6,7 @@ \chapter{Introduction}
 
 \begin{figure}
   \centering
-  \includegraphics{frog}
+  \imageKeepAspect{frog}
```
Comment (ibnesayeed) [inline chapters/01_introduction.tex]: Please keep the basic name `\includegraphics` in the sample. We will document various macros in the documentation. If you really want to highlight the difference, you can add another figure block. illustrating its benefits (you can reuse the image though).

**Review by ibnesayeed — CHANGES_REQUESTED** [review]

File: custommacros.sty (hunk 1 above)
Comment (N0taN3rd) [reply custommacros.sty]: Yes and I have corrected the type

File: chapters/01_introduction.tex (hunk 2 above)
Comment (N0taN3rd) [reply chapters/01_introduction.tex]: Reverted back to original

**Pull request merged**